### ISSUE TITLE
Does not work with Jet

### ISSUE THREAD

**Issue by crazydog115** [codeship/shipscope#28]
Jet builds do not appear in Shipscope. The last build seen for a repo using Jet is the last non-Jet build.

**Comment by mcgaffin** [comment]
Yes, this is true. As far as I know Codeship does not support Jet with its API. I am no longer associated with Codeship, so I couldn't tell you if they have plans to provide a public API for Jet.
